connector_manager/Makefile

```diff
@@ -7,12 +7,14 @@ help: ## help message, list all command
 
 .PHONY: run dev
 run dev: ## run the job
-	python3 -m manager
+	. .venv/bin/activate && python3 -m manager
 
 .PHONY: test
-test: ## run the test
-	python3 -m pytest
+test: ## run the test and activate venv
+	. .venv/bin/activate && pytest -v --cov=manager --cov-report=term-missing
```

Comment:
We definitely should switch to uv soon. Not necessarily in this pull request, just in general. Makes that stuff go away.